lab-cpp_5-10_main.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>		// Підключення стандартної бібліотеки воду/виводу для коректної роботи може знадобитися додатковий ключ під час компіляції -lstdc++
+#include "lab-cpp_5-10_class.h"	// Підключення власної бібліотеки з класами для обєктів файл і папка (методи і фукції додаються на моменті компіляції)
```

Comment:
reading name of heading, I have to imagine what's inside. "lab-cpp_5-10_class" say to me nothing